### DISCUSSION TITLE
I can't push my code to the github

### DISCUSSION THREAD

**Discussion by mastercode1010** [community/community#134432] [GitHub Education] [answered]
### Body

I already used my github account but it's suspended now. So I created new account and reset the config settings using my new account. But it's not working now. How can I solve this problem?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by aimaster-dev** [comment, marked as answer]
Well, I can't see why this happens but I think you didn't complete your resetting git account in your pc. You need to delete your credential in Credential Manager.
![Screenshot 2024-08-02 101928](https://github.com/user-attachments/assets/f3bef146-e516-4520-925c-7735bbbde47b)
I've attached the screenshot associated with this.

**Reply by mastercode1010** [reply]
So you mean, my credential isn't deleted now?

**Reply by aimaster-dev** [reply]
Maybe. You can check this.

**Reply by mastercode1010** [reply]
Great. Thanks.
